### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by cassie365**
Just updated the README.md file. No code changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,63 @@
...
+- [H2 1.4.200](https://www.h2database.com/html/main.html) Using PostgreSQL9 as the dialect
+- [Hibernate 4.3.11.Final](https://docs.jboss.org/hibernate/orm/4.3/manual/en-US/html_single/)
+- [Spring Boot 2.4.1](https://docs.spring.io/spring-boot/docs/2.4.1/reference/html/using-spring-boot.html#using-boot-build-systems)
+- [Java 1.8](https://javaee.github.io/javaee-spec/javadocs/)
+- [JUnit 5.7.0](https://junit.org/junit5/docs/snapshot/release-notes/#release-notes-5.7.0)
+* * *
 
-Ronald Martz
+## Getting Started / Contributing
+The below instructions are intended to be used with [IntelliJ IDEA Community Edition](https://www.jetbrains.com/idea/download/#section=windows) If additional instrucations are necessary, please reach out.
```
Comment (RevatureGentry) [inline README.md]: 💯

File: README.md [hunk 2]
```diff
@@ -1,7 +1,63 @@
...
+If you see accessors/mutators in the structure you are good to go!
+
+5) If you continue to get issues, try going to 
+*File->Settings->Build, Execution, Deployment->Compiler->Annotation Processors*
+and check 'Enable annoation processing'
+
+8) If all else fails, please reach out.
+
+### Requirements
+- Tests first, please push and contact William to approve before moving forward
```
Comment (RevatureGentry) [inline README.md]: Could you change this to be something like "open a Pull Request, adding @RevatureGentry or @KennethDavis391 and your trainer as approvers".
We'll get email notifications when this happens, no need to directly contact 👍

**Review by RevatureGentry — CHANGES_REQUESTED** [review]
LGTM for the most part.
Does everyone in the batch use IntelliJ Community? If anybody uses Eclipse/STS/VS Code, let's add setup for those editors as well.

File: README.md (hunk 2 above)
Comment (cassie365) [reply README.md]: @RevatureGentry Hey Will! I updated the documentation as suggested. Let me know if you notice anything else we should add!

**Review by RevatureGentry — APPROVED** [review]
LGTM 👍

**Pull request merged**